packages/core/src/Button/index.tsx

```diff
@@ -497,3 +498,174 @@ export const IconButton = React.forwardRef<BaseElement, IconButtonProps>(
     )
   }
 )
+
+/**
+ * IconTextButton
+ *
+ * Always has a primary icon and a secondary text
+ *
+ */
+
+export const NativeIconTextButton = styled.button<{
+  readonly icon: IconType
+  readonly visibleFocus: boolean
+}>`
+  white-space: nowrap;
+  max-width: 100%;
+  height: ${componentSize.small};
+  outline: none;
+  &::-moz-focus-inner {
+    border: 0;
+  }
+  border: none;
+  border-radius: ${shape.radius.small};
+  cursor: pointer;
+  user-select: none;
+  transition: all 200ms;
```

Comment:
These code is same as `NativeButton`. So maybe it's better that we extract common style somewhere? 
ex.
```
const commonButtonStyleOrSomething = css`
  white-space: nowrap;
  ....
`
// And use it in `NativeButton` and in `NativeIconTextButton` ?
${commonButtonStyleOrSomething}
```

Reply:
Common CSS  for `NativeButton` and  `NativeIconTextButton` extracted.